maven-scm-providers/maven-scm-provider-jazz/src/main/java/org/apache/maven/scm/provider/jazz/command/tag/JazzTagCommand.java

```diff
@@ -176,7 +176,7 @@ public JazzScmCommand createTagCreateSnapshotCommand( JazzScmProviderRepository
             command.addArgument( message );
         }
 
-        command.addArgument( repo.getRepositoryWorkspace() );
```

Comment:
What is the difference between those two?

Reply:
repo.getRepositoryWorkspace() is a value configured in pom.xml.
repo.getWorkspace() is the current workspace ("branch" in jazz terminology).
So, with the first value, if you switch branch, do some checkins and tag, the tag is created somewhere else.